cm_sdk/models/__init__.py

```diff
@@ -0,0 +1,266 @@
+from enum import Enum
+import json
+from json import JSONEncoder
+import re
+import logging
+
+__author__ = 'e0d'
+
+
+class StringUtil(object):
```

Comment:
I see this in the case of StringUtils, but here I think they are directly related to the this specific class and belong with it.